rulebook.tex

```diff
@@ -1364,32 +1364,41 @@ \subsubsection{Production Plan}
 requiring 1 base, and two colors requiring no additional base at
 all. Products retrieved from the Storage Station may be used only for
 $C_0$ orders with a requested quantity equal to or greater than two.
-
-In each game, up to 10 orders will be placed.\footnote{There will up
-  to 9 orders in regular game time, and an additional order in the
-  case of overtime. The number can be adjusted during the tournament
-  by the Technical Committee, should this turn out to be necessary.}
+An order can be \emph{competitive} or \emph{non-competitive}. For a
+competitive order, only the team that delivers first will score points
+for the delivery, and it will score twice the amount of points. The
```

Comment:
But on the other hand, if the team scores the same amount of points, the incentive to pursue a competitive order is rather low, because it is more risky and offers no benefit other than blocking the other team from scoring those points. This may be sufficient if there are a number of competitive orders, but if we limit them, then the other team may also just decide to ignore it.

I think pursuing a competitive order should have a benefit other than blocking the other team from scoring.

Reply:
The incentive is to "steal" the points from the other team. But I do see your point that at the current state some (most?) teams will simply pursue their guaranteed orders.

I think what I really dislike is a "factor" of points. We do not have this anywhere. If we keep this, let's make this a static number of points, e.g., 10 points on successful (accepted) competitive delivery. In the refbox, there is currently no notion of factors, just summed points. I don't think we need to account for late delivery for the competitive points addition, if you deliver and score at all, you get 10 points (or so) extra.

Reply:
The idea of the factor was that it makes the rule independent of the actual scoring scheme for the non-competitive variant. But I think we can also rephrase this as only giving additional points for the delivery of a competitive order, and no points for delivery if it is the second delivery for that order.

This would reduce the penalty for a second delivery for a competitive complex order, as the team would still get points for the intermediate steps, just not for the delivery anymore.

Reply:
We could also remove the time limit on the competitive order, i.e., make it a standing order. This would also fit with the real-world scenario in mind, where the manufacturer just wants to get the product as fast as possible, but without a specified deadline. What do you think?

This would make the scoring a lot simpler.

Reply:
I got rid of the factor and assigned +20 for the first delivery, which doubles the delivery points for a regular delivery.

Reply:
This is too much in my opinion. It is given too much emphasis on the competitive order. As I stated earlier, let's start slowly and see how it goes. 10 seems to be enough for me, that's ~30% of the overall product you get on top in addition of the fact that a competing team might just have lost work effort worth 30 points if they were slower.

Making it a standalone order makes it less of a strategic run-time decision, I'd keep it with a time window (in overtime the time window would span the entire overtime).

Reply:
I reduced the bonus to 10 points.